exasol_script_languages_container_ci_setup/lib/aws/wrapper/codebuild_service.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, Any, List, Callable
+
+from exasol_script_languages_container_ci_setup.lib.aws.wrapper.datamodels.codebuild import BuildBatch
+from exasol_script_languages_container_ci_setup.lib.aws.wrapper.datamodels.common import PhysicalResourceId
+
+
+class CodeBuildService:
+    def __init__(self, boto_client):
+        self._boto_client = boto_client
+
+    @property
+    def boto_client(self) -> Any:
```

Comment:
You are returning `Any` so is it relevant that it is named `boto_client` rather than just `client`?